src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -99,13 +90,20 @@ const GiftPopover = (
   </Popover>
 );
 
-const navBarScheduleClassName = "NavBar__schedule-dropdown";
+const VolumePopover = (
+  <Popover
+    id="volume-popover"
+    className="NavBar__volume-popover NavBar__volume-popover--without-sliders"
+  >
+    <Popover.Content>
+      <VolumePopOverContent />
+    </Popover.Content>
+  </Popover>
+);
 
-export interface NavBarPropsType {
-  hasBackButton?: boolean;
-}
+const navBarScheduleClassName = "NavBar__schedule-dropdown";
```

Comment:
If just a static string, inline it with its component to be at the place of usage. For the purpose of targeting elements hooks like `useRef` can help in order to comply with the idiomatic React use